cmd/cluster/etcd_health.go

```diff
@@ -0,0 +1,239 @@
+package cluster
+
+import (
+	"bytes"
+	"context"
+	"flag"
+	"fmt"
+	"os"
+	"path/filepath"
+
+	operatorv1 "github.com/openshift/api/operator/v1"
+	"github.com/spf13/cobra"
+	corev1 "k8s.io/api/core/v1"
+	"k8s.io/cli-runtime/pkg/genericclioptions"
+	"k8s.io/client-go/kubernetes"
+	"k8s.io/client-go/kubernetes/scheme"
+	"k8s.io/client-go/rest"
+	"k8s.io/client-go/tools/clientcmd"
+	"k8s.io/client-go/tools/remotecommand"
+	"k8s.io/client-go/util/homedir"
+	cmdutil "k8s.io/kubectl/pkg/cmd/util"
+	"sigs.k8s.io/controller-runtime/pkg/client"
+)
+
+const (
+	EtcdNamespceNAME        = "openshift-etcd"
+	EtcdMemberConditionType = "EtcdMembersAvailable"
+	ConatinerName           = "etcdctl"
```

Comment:
Just some minor typo nits (will also need fixing wherever used)
```suggestion
	EtcdNamespaceName        = "openshift-etcd"
	EtcdMemberConditionType = "EtcdMembersAvailable"
	ContainerName           = "etcdctl"
```

Reply:
This has been addressed in my latest commit.

Reply:
This has been addressed in my latest commit.